app/bundles/FormBundle/Model/SubmissionModel.php

```diff
@@ -249,7 +249,7 @@ public function saveSubmission($post, $server, Form $form, Request $request = nu
                 }
             }
 
-            if (empty($value) && $f->isRequired()) {
+            if ($value === '' && $f->isRequired()) {
```

Comment:
Not sure if we want this behaviour?

Reply:
I think this was changed by another PR sometime back because a valid value could be 0